css/styles.css

```diff
@@ -0,0 +1,137 @@
+  a:hover {
+    text-decoration: underline; }
+
+/* ================================================== Header ======================================================== */
+header {
+  background-image: url("https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg");
+  background-position: 50% 100%;
+  width: 100%; }
+  header nav {
+    margin-top: 80px;
+    margin-bottom: 80px;
+    font-size: 13px; }
+    header nav ul {
+      border-radius: 5px;
+      background-image: linear-gradient(#969696, #5e5e5e, #6b6b6b);
+      box-shadow: 0 1px 4px #656565;
+      display: flex;
+      width: 80%;
+      margin-left: auto;
+      margin-right: auto;
+      border-top: solid 1px #5d5d5d;
+      border-bottom: solid 1px #5d5d5d;
+      border-right: solid 1px #5d5d5d; }
+    header nav li {
+      display: inline-block;
+      text-align: center;
+      border-left: solid 1px #5d5d5d;
+      width: 15%;
+      padding: 8px; }
+      header nav li:hover {
+        background: linear-gradient(#646464, #434343, #5a5a5a); }
+      header nav li a {
+        text-decoration: none;
+        color: #ffffff;
+        text-align: center; }
+      header nav li form {
+        display: flex;
+        width: 100%; }
+      header nav li label {
+        display: inline-block;
+        background: #ffffff;
+        padding: 1px;
+        width: 15%; }
+      header nav li .left-label {
+        padding-left: 3px;
+        padding-right: 5px;
+        border-top-left-radius: 50%;
+        border-bottom-left-radius: 50%;
+        font-size: 12px; }
+      header nav li input {
+        display: inline-block;
+        padding: 1px;
+        width: 80%;
+        border-top-right-radius: 10px;
+        border-bottom-right-radius: 10px; }
+    header nav .logo-icon {
+      font-size: 20px;
+      padding: 2px; }
+  header div {
+    margin-top: 30px;
+    margin-bottom: 210px; }
+
+/* =================================================== Main ========================================================= */
+main {
+  width: 100%;
+  display: flex; }
+  main .articles {
+    width: 25%;
+    height: 200px;
+    overflow: hidden; }
+    main .articles a {
+      height: 100%;
+      display: flex; }
+    main .articles img {
+      width: 100%; }
+
+/* ==================================================== Footer ====================================================== */
+footer {
+  display: flex;
+  width: 100%;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  font-size: 10px;
+  padding: 20px 30px 0 30px; }
+  footer div {
+    width: 40%; }
+  footer .left-div {
+    margin-left: 100px;
+    color: #949494; }
+    footer .left-div p {
+      margin-bottom: 15px; }
+  footer .right-div {
+    margin-right: 100px; }
+  footer ul {
+    display: flex;
+    list-style: none; }
+  footer li a {
+    display: inline-block;
+    padding-right: 6px;
+    padding-left: 6px; }
+  footer a {
+    color: #3992d1; }
+
```

Comment:
`HINT`

Please, fix the indentation. :blush:
A good indentation is fundamental for good readability of the code.
Many IDE  can help you with this task.
This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

![Screenshot of how to format code using VS Code](https://res.cloudinary.com/bolaah/image/upload/v1563045186/github-microverse-project/code-reviews/code-review002.gif)